src/core/__mocks__/ad-sizes.ts

```diff
@@ -0,0 +1,21 @@
+import { AdSize as RealAdSize } from 'core/ad-sizes';
+
+class AdSize extends RealAdSize {
```

Comment:
There's a jest error that pop's up now and again when trying to assert on instances of `AdSize`, this resolves it. I think jest tries to copy this by using `new AdSize(length)` which won't work.